db/migrate/20190902133711_add_tags_links_digest_to_existing_subscriber_lists.rb

```diff
@@ -0,0 +1,10 @@
+class AddTagsLinksDigestToExistingSubscriberLists < ActiveRecord::Migration[5.2]
+  def change
+    SubscriberList.all.each do |list|
```

Comment:
You'd probably want to do a find_each here so you don't load the entire query in memory.